packages/web-components/src/components/va-process-list/va-process-list-item.tsx

```diff
@@ -57,11 +57,15 @@ export class VaProcessListItem {
     
     return (
       <Host role="listitem" class='usa-process-list__item'>
-        { status ?
-          <div class="usa-process-list__heading-eyebrow">{statusTextMap[status]}</div>
-          : null
-        }
-        {header ? <HeaderLevel part="header" class='usa-process-list__heading'>{header}</HeaderLevel> : null}
+        {header || status ? (
+          // aria-label hack to avoid header being read twice in voiceOver
+          <HeaderLevel part="header" class='usa-process-list__heading' aria-label={`${status ? `${statusTextMap[status]}: ` : ''}${header}`}>
```

Comment:
In VoiceOver when there are multiple elements in the header it will read it as "Header level 3; two items;" and repeat the heading text multiple times. The `aria-label` in the header and `aria-hidden`  on the container fixes this issue, but it seems like there should be a better solution? @rsmithadhoc do you have a suggestion?

Reply:
I tested in other screen readers, JAWS didn't recognize the headings at all. I'll take a look and see if there's a better solution.

Reply:
@powellkerry 
I think the original HTML may be the way to go here.
```
    <h2>
        <span></span>
        <span></span>
    </h2>
```

When reading the page, I see that VoiceOver reads the heading, then repeats the 2 items within the heading, like you said. But when using the command to read the next heading it only reads it once and using the headings menu it only reads it once, as expected.

While not ideal for VoiceOver, we may have to accept it for now. It may be confusing, but I don't think it would hinder the user in navigating the page. JAWS and NVDA only read it once when reading the page. I also tested them reading next heading or using their headings menu and it also functioned as intended with the original HTML.

I think the HTML is fine in the original way, so VoiceOver _should_ be able to handle it. I think it also keeps things much simpler and should work in a wider range of places, as `aria-hidden` can cause some issues, like I saw with JAWS.

Thanks!